data/human/human missions.txt

```diff
@@ -7453,3 +7453,443 @@ mission "Adelita 5"
+			`	Howard's smile falters slightly. He forces a light chuckle, though the tension is noticeable. "Well, if I'm being honest, the Sunracer office is... also technically reservable," he admits, rubbing his neck and looking almost pained. "But just as nice! And, uh... look, my real office is technically a cubicle, but compared to the kiosk last time we met? It's a big step up." He shrugs, trying to brush it off. "But hey, we don't need fancy spaces to make serious deals. Let's get back on track, yeah?" He clears his throat.`
+			choice
+				`	"Absolutely."`
+					goto next2
+				`	"Howard, a promotion's a big deal. The office doesn't matter--your work speaks for itself."`
+					goto kindy
+				`	"Howard, your cubicle is still pathetic. Now, let's get back on track."`
+			action
+				"syndicate business: mean to Howard" ++
+			`	Howard flinches, his jaw tightening for a moment before he forces a thin smile. "Pathetic or not, it gets the job done," he says quietly, avoiding your gaze as he fidgets with a paperweight on his desk. "Spaceport. Later." His voice is more clipped now, and he doesn't wait for a response before focusing on his work calls, shutting you out entirely.`
+				decline
+			label kindy
+			action
+				"syndicate business: mean to Howard" --
+			`	"Thanks, that means a lot. Now, I've got to get back to work."`
+			label next2
+			`	Howard's attention shifts away from you as he begins taking a series of work calls, his tone changing seamlessly from charming to authoritative as he negotiates and makes deals. He mouths two words to you: "Spaceport... later..." It's clear that, for now, Howard is finished with you and you'll get no more information.`
+
+
+mission "Syndicate Business 4"
+	name "Escort Howard to <planet>"
+	description "Escort Howard from Megaparsec Inc. to <planet> to learn what favor is needed in exchange for an investment opportunity."
+	priority
+	source "Hephaestus"
+	to offer
+		has "Syndicate Business 3: done"
+	destination "Sunracer"
+	on offer
+		conversation
+			`You meet Howard at the spaceport. He's standing on a nearby landing pad by his Shuttle, dressed sharply but looking a bit hurried. He gives you a quick nod as you approach.`
+			`	"Good to see you're on time," Howard says, glancing at his antique wristwatch. "We've got a lot of ground--or space--to cover." Howard briefly looks at the <ship>. "Your ship ought to go fast enough, I suppose. I'll be tied up in calls for most of the trip, but once we're back at the office, we can talk." He starts toward his Shuttle's ramp, already shifting his attention back to his wristwatch. "Let's get moving."`
+				launch
+	npc accompany save
+		government "Syndicate"
+		personality escort timid
+		ship "Shuttle" "Gallant Knight"
+		on kill
+			dialog "You watch as Howard's Shuttle explodes into a cloud of debris along with any hope of an investment opportunity."
+	npc
+		government "Pirate"
+		personality harvests nemesis plunders staying
+			confusion 40
```

Comment:
These are very amateur pirates.

Reply:
I know pirates are typically 20, for this one I copied the confusion used for Tiny Escort jobs. These fellas aren't likely to be more than a minor nuisance regardless of confusion, so happy to change if that's desired.